corehq/apps/es/README.rst

```diff
@@ -318,6 +318,36 @@ Using this adapter in practice might look as follows:
     classic_book = books_adapter.fetch("978-0345391803")
 
 
+Tombstone
+'''''''''
+
+The concept of Tombstone in the ES mulitplexer is there to be placeholder for the docs that are deleted in the primary index. It means that whenever an adapter is multiplexed and a document is deleted from the primary index, then the secondary index will create tombstone entry for that document. The python class defined to represent these tombstones is  `corehq.apps.es.client.Tombstone`
+
+The requirement of tombstones arise due to a potential race condition that might come when reindex operation is run on the multiplexed index.
+
+Scenario without tomstones: If a multiplexing adapter deletes a document in the secondary index (which turns out to be a no-op because the document does not exist there yet), and then that same document is copied to the secondary index by the reindexer, then it will exist indefinitely in the secondary even though it has been deleted in the primary.
```

Comment:
How can the reindexer copy the doc to the secondary if it does not exist in the primary? I think we could remove this paragraph and just go with the bullet points below.